### PR TITLE
feat: Add lables for route/consumer/service/ssl objects

### PR DESCRIPTION
**Pull request by imjoey**
Signed-off-by: imjoey <majunjiev@gmail.com>

### What this PR does / why we need it:
Following with #2279 , this PR is going to add `labels` property support for `Route`/`Consumer`/`Service`/`SSL` objects, as well as fix some typos introduced by #2279 .

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible?

### CODE REVIEW

**Comment by imjoey** [comment]
ping @membphis @nic-chen @gxthrj for reviewing, much appreciated.

File: apisix/schema_def.lua [hunk 1]
```diff
@@ -555,6 +580,14 @@ _M.ssl = {
             type = "integer",
             minimum = 1588262400,  -- 2020/5/1 0:0:0
         },
+        labels = {
```
Comment (nic-chen) [inline apisix/schema_def.lua:583]: I think ssl may not need labels

**Comment by membphis** [comment]
nice PR. LGTM

File: t/admin/consumers.t [hunk 2]
```diff
@@ -223,3 +223,78 @@ GET /t
...
+                         "env":"production",
+                         "version":"v2"
+                     }
+                }]],
+                [[{
+                    "node": {
+                        "value": {
+                            "username": "jack",
+                            "desc": "new consumer",
+                            "labels": {
```
Comment (liuxiran) [inline t/admin/consumers.t:250]: node does not support `labels`, so here may need to remove `labels`?

File: apisix/schema_def.lua (hunk 1 above)
Comment (imjoey) [reply apisix/schema_def.lua:583]: @nic-chen  IMHO, we could use label to distinguish among dozens of SSLs according to their usage.

Comment (imjoey) [reply apisix/schema_def.lua:583]: @membphis  @liuxiran  any thoughts on this? 😄

Comment (membphis) [reply apisix/schema_def.lua:583]: > we could use label to distinguish among dozens of SSLs according to their usage.

Agree this. Searching by category will be helpful.

Comment (liuxiran) [reply apisix/schema_def.lua:583]: > we could use label to distinguish among dozens of SSLs according to their usage

agree +1

Comment (nic-chen) [reply apisix/schema_def.lua:583]: OK, I think it could work with SNI.But for management, labels is helpful for it.

**Review by membphis — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**